client/my-sites/domains/domain-management/list/domain-item.jsx

```diff
@@ -313,7 +315,7 @@ class DomainItem extends PureComponent {
 			} );
 		}
 
-		if ( domainDetails?.emailForwardsCount > 0 ) {
+		if ( hasEmailForwards( domainDetails ) ) {
 			return translate( '%(emailForwardsCount)d forward', '%(emailForwardsCount)d forwards', {
 				count: domainDetails.emailForwardsCount,
```

Comment:
Maybe we should also use `getEmailForwardsCount()` here and two lines below?

Reply:
Done!